### PR TITLE
Extract messages of unsupported exception as constants

### PR DESCRIPTION
**Pull request by zyfx595701088**
Optimize code

### CODE REVIEW

File: src/main/java/redis/clients/jedis/util/RedisInputStream.java
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/util/RedisInputStream.java:1]: Please don't change anything out of the scope of the PR. If you want you can do it in separate PR.

**Review by sazzad16 — CHANGES_REQUESTED** [review]

File: src/main/java/redis/clients/jedis/util/RedisInputStream.java
Comment (zyfx595701088) [reply src/main/java/redis/clients/jedis/util/RedisInputStream.java:1]: ok, I will do it.
